### PR TITLE
Drop prerelease version label

### PR DESCRIPTION
**Pull request by MichaelSimons**
Fixes #https://github.com/dotnet/source-build/issues/4670.

Second attempt of https://github.com/dotnet/source-build-reference-packages/pull/1070

[Test Build](https://dev.azure.com/dnceng/internal/_build/results?buildId=2589920&view=results) (internal Microsoft link)

### CODE REVIEW

File: eng/Versions.props [hunk 1]
```diff
@@ -3,8 +3,8 @@
     <MajorVersion>10</MajorVersion>
     <MinorVersion>0</MinorVersion>
     <PatchVersion>0</PatchVersion>
-    <PreReleaseVersionLabel>alpha</PreReleaseVersionLabel>
     <PreReleaseVersionIteration>1</PreReleaseVersionIteration>
```
Comment (mmitche) [inline eng/Versions.props:6]: You should remove the iteration too (it won't get included since the version label is empty)

**Review by mthalman — COMMENTED** [review]
What's been changed that will make this work now?

**Comment by MichaelSimons** [comment]
> What's been changed that will make this work now?

https://github.com/dotnet/source-build-reference-packages/pull/1086/files#diff-1ea18ff65faa2ae6fed570b83747086d0317f5e4bc325064f6c14319a9c4ff67R7

**Review by mthalman — APPROVED** [review]

File: eng/Versions.props (hunk 1 above)
Comment (MichaelSimons) [reply eng/Versions.props]: Fixed in https://github.com/dotnet/source-build-reference-packages/pull/1086/commits/546f721a8c7c3f16dfb0edf489955c4d47e6eb61

**Pull request merged**